.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+# .github/workflows/main.yml
+
+name: CI
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  test-ruby:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ubuntu]
+        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
```

Comment:
There's no harm in supporting these older versions (looks like the gemspec doesn't state a hard lowest ruby version dep?), but just wanted to note:

- Ruby 3.0 and earlier are EOL
- Maybe add 3.3 to this matrix?